easybuild/easyconfigs/c/Cython/Cython-0.26-GCCcore-6.4.0-Python-2.7.13-bare.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '0.26'
+pyver = '2.7.13'
+pyshortver = '2.7'
+pysubver = '-bare'
+versionsuffix = '-Python-%s%s' % (pyver, pysubver)
```

Comment:
Correct... the templates don't work when it is a builddep.

The point here is avoid building Cython for every toolchain option.  e.g. for Python-2.7.13I don't want/need three different versions (intel/iomkl/foss) when one will do.   Same for 3.6.2.   

I need different ones for 2.7 and 3.6 so that when PythonPackage bite-compiles the .py (to create .pyc) files it uses the right Python and sets PYTHONPATH appropriately.

Python has to be a build dep at this stage (Cython is only a library and serves no real purpose on its one), since it will be later used by something that has the full-blown Python (with MPI/MKL).  That is, unless you have some way to make the GCCcore-bare version not conflict with the full-blown version (which you don't).